docs/plan/fix-ec-payment-event-on-browser-back.md

```
@@ -0,0 +1,567 @@
+
+### Changes Required:
+
+#### 1. Nový hook
+**File**: `project-base/storefront/utils/app/usePaymentReturnHandler.ts`
+**Changes**: Nový soubor
+
+```typescript
+import { useRouter } from 'next/router';
+import { useEffect, useRef } from 'react';
+import {
+    getGtmPendingPaymentFromLocalStorage,
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)